plot.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+from matplotlib.ticker import MultipleLocator
```

Comment:
Please add an explanation (docstring) at the top that describes what this file is doing.

Reply:
This part is still pending.